PyRoute/Calculation/TradeCalculation.py

```diff
@@ -175,10 +190,12 @@ def get_trade_between(self, star, target):
             "This route from " + str(star) + " to " + str(target) + " has already been processed in reverse"
 
         try:
-            route = astar_path(self.galaxy.stars, star, target, self.galaxy.heuristic_distance)
+            route, diag = astar_path(self.galaxy.stars, star, target, self.galaxy.heuristic_distance)
```

Comment:
Return performance diagnostics for pathfinding - number of:
nodes expanded
nodes queued
neighbours checked
heuristic calls

to better guide choice of pathfinding algorithm in later work